src/Microsoft.Azure.SignalR.Common/Interfaces/IMultiEndpointServiceConnectionContainer.cs

```diff
@@ -0,0 +1,12 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.SignalR
+{
+    internal interface IMultiEndpointServiceConnectionContainer : IServiceConnectionContainer
+    {
+        public IReadOnlyDictionary<ServiceEndpoint, IServiceConnectionContainer> ConnectionContainers { get; }
```

Comment:
why need to expose the property? maybe exposing method like `GetContainer(ServiceEndpoint endpoint)` is safer

Reply:
Only used in tests. Currently `IMultiEndpointServiceConnectionContainer` is not directly used by outside `MultiEndpointServiceConnectionContainer`, remove the property is good.